ads/google/a4a/utils.js

```diff
@@ -78,21 +78,28 @@ export const TRUNCATION_PARAM = {name: 'trunc', value: '1'};
 /**
  * Check whether Google Ads supports the A4A rendering pathway is valid for the
  * environment by ensuring native crypto support and page originated in the
- * the {@code cdn.ampproject.org} CDN <em>or</em> we must be running in local
+ * {@code cdn.ampproject.org} CDN <em>or</em> we must be running in local
  * dev mode.
  *
  * @param {!Window} win  Host window for the ad.
  * @returns {boolean}  Whether Google Ads should attempt to render via the A4A
  *   pathway.
  */
 export function isGoogleAdsA4AValidEnvironment(win) {
-  const supportsNativeCrypto = win.crypto &&
-      (win.crypto.subtle || win.crypto.webkitSubtle);
   const googleCdnProxyRegex =
       /^https:\/\/([a-zA-Z0-9_-]+\.)?cdn\.ampproject\.org/;
```

Comment:
Are we consisted about the domain cdn.amproject.org.foo.com?  Seems like we should guard against that.